### PR TITLE
Updated hun

### PR DESCRIPTION
**Pull request by frodo19**

### CODE REVIEW

**Comment by mediaminister** [comment]
Thanks for the translation.

File: resources/language/resource.language.hu_hu/strings.po [hunk 1]
```diff
@@ -273,35 +273,35 @@ msgstr "Megpróbáljuk újra?"
...
 
 msgctxt "#30821"
 msgid "[B]Widevine CDM[/B] is at version [B]{version}[/B] and was installed on [B]{date}[/B]"
-msgstr ""
+msgstr "[B]Widevine CDM[/B] verzió [B]{version}[/B] telepítve [B]{date}[/B]"
 
 msgctxt "#30822"
 msgid "It was extracted from Chrome OS image [B]{name}[/B] with version [B]{version}[/B]"
-msgstr ""
+msgstr "Legutóbb kicsomagolva a Chrome OS image-ból [B]{name}[/B] verzió [B]{version}[/B]"k
```
Comment (horstle) [inline resources/language/resource.language.hu_hu/strings.po:296]: I think the k at the end should be removed?

Comment (mediaminister) [reply resources/language/resource.language.hu_hu/strings.po:296]: I saw it after merging. Will fix it.

Comment (frodo19) [reply resources/language/resource.language.hu_hu/strings.po:296]: Yes. sorry, updated again

**Pull request merged**